src/components/screens/public/LoginProvider/LoginProvider.interactor.vue

```diff
@@ -1,21 +1,17 @@
 <template>
-  <loading-screen :is-loading="isLoading">
```

Comment:
It's not clean interactor. It's bad practice to rename with new approach and to stay with old inner content. We will see structure and thinking that it has already been refactored